src/components/Repos/Repos.js

```diff
@@ -1,56 +1,39 @@
 import './Repos.css'
-import { Link } from "react-router-dom";
+import { Link, useParams } from "react-router-dom";
 import { useState, useEffect } from 'react'
 
-// show all repositories for me on the page by default when i load the page.
-// show also a input field where i can type any user's github username
-// when i hit the button, i want to see their repositories on the page.
-
-// clicking a repo should navigate to /user/repo-name and show information about the repo
-// I should be able to navigate back to the home page showing a list of repositories for the user
-
-const initialFormData = {
-  github: 'dearshrewdwit'
-}
-
 function Repos () {
   const [repos, setRepos] = useState([])
-  const [username, setUsername] = useState('dearshrewdwit')
-  const [formData, setFormData] = useState(initialFormData)
+  const [user, setUser] = useState({})
   const [notFound, setNotFound] = useState(false)
+  const { username } = useParams()
```

Comment:
How come you initialised your state in Repos.js? Might it make more sense to do this in App.js and and pass these variables down through components?